framework/basehandlers.py

```diff
@@ -13,44 +13,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from datetime import datetime
 import json
 import logging
 import os
 import re
+from datetime import datetime
 from typing import Any, NoReturn, Optional
 
 import flask
 import flask.views
-import werkzeug.exceptions
-
 import google.appengine.api
+import werkzeug.exceptions
+from flask import render_template, session
+from flask_cors import CORS
 from google.cloud import ndb  # type: ignore
 
 import settings
 from api import api_specs
-from framework import csp
-from framework import permissions
-from framework import secrets
-from framework import users
-from framework import utils
-from framework import xsrf
-from internals import approval_defs
-from internals import notifier_helpers
-from internals import user_models
+from framework import csp, permissions, secrets, users, utils, xsrf
+from internals import approval_defs, notifier_helpers, user_models
```

Comment:
Is there a reason for updating the imports like this? This does not hurt the functionality, but it is not the typical import structure used for the project

Reply:
seems like it runs `organize imports`. I'll revert it. thanks